README.md

```diff
@@ -25,269 +25,221 @@
-to schedule jobs per second (ie: `30 * * * * *` would fire a job every 30
-seconds past the minute).
+Cron, every, and interval types push jobs into sidekiq in a recurrent manner.
 
-A big shout out to [rufus-scheduler](http://github.com/jmettraux/rufus-scheduler)
-for handling the heavy lifting of the actual scheduling engine.
+`cron` follows the same pattern as cron utility, with seconds resolution.
 
-NOTE: As schedules are persisted in Redis, an empty schedule will purge any schedule previously stored.
-sidekiq-scheduler instances running with `dynamic` flag set to `true` and empty schedule configuration will not push any job.
-Before version 2.0.8, instances matching those conditions read the previously stored schedule and push the jobs to sidekiq as usual.
+``` yaml
+:schedule:
+  HelloWorld:
+    cron: '0 * * * * *' # Runs when second = 0
+```
 
-With version >= `2.0.8` the purge of Redis schedules config can be prevented by setting
-`enabled` flag to `false`:
+`every` triggers following a given frequency:
 
-```yaml
-:enabled: false
+``` yaml
+    every: '45m'    # Runs every 45 minutes
 ```
-### Enabling/Disabling jobs
 
-By default all the jobs are enabled. There is an option at job's configuration to change
-this behavior.
+`interval` is similar to `every`, the difference between them is that `interval` type schedules the
+next execution after the interval has elapsed counting from its last job enqueue.
 
+At, and in types push jobs only once. `at` schedules in a point in time:
 ``` yaml
-clear_leaderboards_moderator:
-  every: "30s"
-  class: CheckDaemon
-  enabled: false
-  description: "This job is disabled by default"
+    at: '3001/01/01'
 ```
 
-However, the flag can be changed through the web interface at the Recurring Jobs tab.
+You can specify any string that `DateTime.parse` and `Chronic` understand. To enable Chronic
+strings, you must add it as a dependency.
 
-NOTE: Changes made thorugh the UI supersede job's configuration, even after a restart.
+`in` triggers after a time duration has elapsed:
 
-### Loading the schedule
+``` yaml
+    in: 1h # pushes a sidekiq job in 1 hour, after start-up
+```
 
-Let's assume your scheduled jobs are defined in a file called `config/scheduler.yml` under your Rails project,
-you could create a Rails initializer called `config/initializers/scheduler.rb` which would load the job definitions:
+You can provide options to `every` or `cron` via an Array:
 
-```ruby
-require 'sidekiq/scheduler'
+``` yaml
+    every: ['30s', first_in: '120s']
+```
 
-Sidekiq.configure_server do |config|
-  config.on(:startup) do
-    Sidekiq.schedule = YAML.load_file(File.expand_path('../../scheduler.yml', __FILE__))
-    Sidekiq::Scheduler.reload_schedule!
-  end
-end
+See https://github.com/jmettraux/rufus-scheduler for more information.
+
+## Load the schedule from a different file
+
+You can place the schedule configuration in a separate file than `config/sidekiq.yml`
```

Comment:
"... place the schedule configuration in a separate file **from** `config/sidekiq.yml`"